### ISSUE TITLE
Visual distinction between a placeholder & input text

### ISSUE THREAD

**Issue by pankajparashar** [captainbrosset/devtools-tips#103]
With placeholder:
<img width="511" alt="image" src="https://github.com/captainbrosset/devtools-tips/assets/38640616/1ebedc72-7820-4856-b295-008931db1d34">

With text:
<img width="511" alt="image" src="https://github.com/captainbrosset/devtools-tips/assets/38640616/c84f947c-ca35-40e2-82ab-afde2ef77497">

Issue is here,
https://github.com/captainbrosset/devtools-tips/blob/main/src/assets/style.css#L249

Do you think the placeholder style should be removed?

**Comment by captainbrosset** [comment]
The difficulty is that the default placeholder color doesn't offer enough contrast with the background color.
But I agree this is a bug and we should find a way to improve this.

Ideally, there should be a visible label too, and not just a placeholder, for accessibility reasons. So maybe the solution is to find a nice way to display a label next to the field, and get rid of the placeholder.

**Comment by pankajparashar** [comment]
Yes, we should definitely have a label. Meanwhile, `rgb(82 82 82)` does offer sufficient contrast ratio to meet WCAG guidelines. I used the color suggested by DevTools to fix the contrast ratio.

![image](https://github.com/captainbrosset/devtools-tips/assets/38640616/1c979080-58b9-4e3b-8b91-1f113dac5c61)
